src/KeyVault/Microsoft.Azure.KeyVault.WebKey/JsonWebKey.cs

```diff
@@ -0,0 +1,637 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+
+namespace Microsoft.Azure.KeyVault.WebKey
+{
+    using System;
+    using System.Linq;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using System.Runtime.Serialization;
+    using System.Security.Cryptography;
+
+    /// <summary>
+    /// As of http://tools.ietf.org/html/draft-ietf-jose-json-web-key-18
+    /// </summary>
+    public class JsonWebKey
```

Comment:
Some probably generated but lots of additions and type is generally usable outside the SDK and we don't want the SDK dependencies. See discussion on Slack re: ms-client-external and the need to have references to externally defined types for a better experience.